### DISCUSSION TITLE
Regional pricing based on Purchasing Power Parity

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#144] [Anything Else]
"I’m a developer from a non-US country. I find it difficult to subscribe to an array of serivces when each one of them charges the same from a US-based employee as they do from those based in countries like India, Brazil, South Africa, Indonesia and so on.

Some privacy-friendly way to verify and offer a cheaper price to residents of not so well off countries, unlike the US, would be very much appreciated."

([imported from the old roadmap](https://plausible.nolt.io/103))

**Comment by metmarkosaric** [comment]
Does anyone have any good examples of how this is done in real life by other startups? Cannot find any examples and seems something that's difficult to organize looking at our current setup, website and how we deal with payments and subscriptions. Thanks!

**Reply by sachinchoolur** [reply]
I faced the same issue with https://www.lightgalleryjs.com/. 

So, I built https://www.paritydeals.com/ that helps you set up PPP pricing with a few clicks by connecting your Stripe account.

ParityDeals create a few coupon codes on your stripe account and display them based on the user's location. 

These coupons are frequently refreshed to avoid unauthorized sharing on social forums. And ParityDesl detects VPNs, proxy servers, and Tor browsing and reverts to original pricing if abuse is detected.

docs - https://www.paritydeals.com/integrations/stripe/

If you want to customize the pricing page instead of showing the automated banner, you can use the API to do that.

API docs - https://www.paritydeals.com/docs/api/deals-discount/

Hope it helps

**Comment by jerodsanto** [comment]
I agree this is a big challenge and, sadly, I don't know of any obvious/public examples off the top of my head. We plan on tackling this as well when we move Changelog++ (our community membership) onsite, so I'm very interested in how y'all go about it.

There was some conversation around this on JS Party when discussing online conferences, which have similar desires.

Here it is: https://changelog.com/jsparty/124#transcript-31

**Comment by metmarkosaric** [comment]
Thanks @jerodsanto!

**Comment by ravivyas84** [comment]
https://stripe.com/docs/api/prices, Check if this works. i.e Setting up a different price for different currencies. 

I have never used Stripe, but I am guessing they are the ones to handle it via the defined currency. 

Edit: Updating the website is a whole different ball game, which I have not looked into, but it does involve javascript :P

**Comment by jerodsanto** [comment]
This is really cool! Here are their supported currencies:

https://stripe.com/docs/currencies

I haven't yet found in the docs if and where they will detect the user's currency for display and charging purposes...

**Comment by metmarkosaric** [comment]
Thanks @ravivyas84! Will explore how Stripe does it and what Paddle can do too.

**Comment by MaybeThisIsRu** [comment]
I had recently paid via Paddle but the workflow used in that scenario was:

1. Site detects IP (probably?)
2. Site offers a discount code specific to my country
3. I start the payment via Paddle for the full amount (say, USD 40)
4. I enter the generic discount coupon code given to me
5. I get the discount and the amount to be paid is USD 15

**If** this is how it was done, this is ripe for abuse, but the author offered her best in good faith regardless. **If** this is the only workflow for Paddle, it might be best to stick to Stripe for intl. customers.

**Comment by MaybeThisIsRu** [comment]
Is this on the agenda any longer?

**Reply by metmarkosaric** [reply]
there's no easy way for us to do this unfortunately so it's not on the agenda at the moment

**Comment by emanoj1** [comment]
Hey,

Perhaps, you've already found the solution by now but if you haven't, I came across these 2 services last week:

1. [Pricery.io](https://www.pricery.io/)
2. [ParityDeals](https://www.paritydeals.com/)

I am educating myself on these 2 products at the moment and hopefully, I can use one of them on my web app project.

Hopefully, this is what you are looking for. If you've managed to get another solution, please do let me know.

All the best.

**Comment by tavlean** [comment]
I recently requested parity pricing to Readwise. They asked for any of my govt issued IDs.
After verification, they added the 50% recurring discount to my account.
